.github/workflows/delete-development-environment.yml

```diff
@@ -0,0 +1,33 @@
+name: Create Development Environment
+on:
+  workflow_dispatch:
+
+run-name: Create development environment for user ${{ github.actor }}
+jobs:
+  run-deploy-entities-to-dev:
+    uses: ./.github/workflows/reusable-deploy.yml
+    with:
+      environment: dev
+      service: entities
+      developer_name: ${{ github.actor }}
+      operation: delete
+    secrets: inherit
+  run-deploy-workflows-to-dev:
+    uses: ./.github/workflows/reusable-deploy.yml
+    with:
+      environment: dev
+      service: workflows
+      developer_name: ${{ github.actor }}
+      operation: delete
+    secrets: inherit
+  #run-deploy-graphql-fed-to-dev:
+  #  uses: chanzuckerberg/czid-graphql-federation-server/.github/workflows/reusable-deploy.yml@rlim/fix-reusable-deploy
+  #  needs: [run-deploy-workflows-to-dev, run-deploy-entities-to-dev]
+  #  with:
+  #    environment: dev
+  #    api_url: http://localhost:3000
+  #    allowed_cors_origin: http://localhost:3000
+  #    nextgen_entities_url:  http://${{ needs.run-deploy-entities-to-dev.outputs.stack_name }}-entities:8008
+  #    nextgen_workflows_url: http://${{ needs.run-deploy-workflows-to-dev.outputs.stack_name }}-workflows:8042
+  #    stack_name: ${{ github.actor }}-fed
+  #  secrets: inherit
```

Comment:
Is this intentionally commented out to use later?

Reply:
yep!